gdc_client/upload/parser.py

```diff
@@ -29,17 +34,18 @@ def upload(parser, args):
     files = read_manifest(args.manifest) if args.manifest else []
 
     if not args.manifest:
-        files.append({
-            'id': args.identifier,
-            'project_id': args.project_id,
-            'path': args.path,
-            'upload_id': args.upload_id,
-        })
+        for uuid in args.file_ids:
+            files.append({
+                'id': uuid,
+                'project_id': args.project_id,
+                'path': args.path,
+                'upload_id': args.upload_id,
+            })
 
     # TODO remove debug - handled by logger
     debug = logging.DEBUG in args.log_levels
 
-    manifest_name = args.manifest.name if args.manifest else args.identifier
+    manifest_name = args.manifest.name if args.manifest else args.file_ids[0]
```

Comment:
does `nargs='*'` for `file_ids` allow 0 args? if so could `file_ids == []` causing an IndexError?